mir_perception/mir_object_recognition/ros/scripts/pc_object_recognizer_node

```diff
@@ -116,11 +113,7 @@ class PointcloudObjectRecognizer():
                         name = "unknown object found"
                         probability = 0.0
                         continue
-
-                    print('\n######################')
-                    print('name: ', name)
-                    print('######################\n')
-
+                    rospy.loginfo('Name of the object found: ',name)
```

Comment:
same here: `rospy.loginfo('Name of the object found: %s',name)`

Reply:
resolved